vllm/attention/backends/ipex_attn.py

```diff
@@ -463,8 +463,13 @@ def forward(
                     value = value.repeat_interleave(self.num_queries_per_kv,
                                                     dim=1)
                 import vllm._C.ops
+                assert self.head_size == 128 or self.head_size == 64
                 value = os.environ.get('USE_CONTEXT_V1')
-                if self.head_size == 128 and value is None:
+                if using_gqa_kernel:
```

Comment:
Maybe we could rename `using_gqa_kernel` to `using_ipexllm_kvcache`?

Reply:
Make sense, will change